### PR TITLE
Translations - Complete Italian + almost complete German translation

### PR DESCRIPTION
**Pull request by mrschick**
**When merged this pull request will:**
- Add all missing Italian translations;
- Add all missing German translations except for `addon/fieldmanual`;
- Review and overhaul all Italian translations;

### CODE REVIEW

**Comment by bux** [comment]
This PR is huge ...

File: addons/arsenal/stringtable.xml [hunk 1]
```diff
@@ -1662,13 +1683,16 @@
...
             <German>Wechseln Sie zwischen der Anzeige aller Elemente oder Ihrer Favoriten.\nDoppelklicken Sie bei gedrückter Umschalttaste, um ein Element hinzuzufügen oder zu entfernen.</German>
+            <Italian>Cambia tra mostrare tutti gli oggetti o solo i tuoi preferiti.\nShift + Doppio Click per aggiungere o rimuovere un preferito.</Italian>
             <Korean>모든 아이템을 표시하거나 즐겨찾기를 표시할 때 전환합니다\nShift 키를 누른 상태에서 두 번 클릭하여 아이템을 추가하거나 제거합니다.</Korean>
             <French>Change entre l'affichage de tous les éléments ou de vos favoris.\nDouble-cliquez en maintenant la touche Maj enfoncée pour ajouter ou supprimer un élément.</French>
             <Portuguese>Alterna entre a exibição de todos os itens ou seus favoritos.\nClique duas vezes enquanto mantém pressionada a tecla Shift para adicionar ou remover um item.</Portuguese>
         </Key>
         <Key ID="STR_ACE_Arsenal_buttonSearchTooltip">
             <English>Search\nCTRL + Click to enable live results</English>
             <Japanese>検索\nCTRL + クリックでライブ結果を有効にします</Japanese>
+            <German>Suche\nSTRG + Click für live-aktualisierung während man schreibt</German>
```
Comment (BaerMitUmlaut) [inline addons/arsenal/stringtable.xml:1694]: Großschreibung, ich würde auch "während des Schreibens" bevorzugen

**Review by BaerMitUmlaut — CHANGES_REQUESTED** [review]

**Review by PabstMirror — APPROVED** [review]

**Review by PabstMirror — APPROVED** [review]

**Comment by PabstMirror** [comment]
@mrschick
I can fix the merge conflicts if you give write access to PR

**Comment by mrschick** [comment]
Unfortunately [not possible on organization PRs](https://github.com/orgs/community/discussions/5634) ATM, will try to resolve them on my end.

**Comment by PabstMirror** [comment]
https://github.com/acemod/ACE3/tree/pr/9673-mergeconflicts has fixes if you can merge that

**Comment by mrschick** [comment]
Ah, just committed the resolution through GitHub's frontend. I guess it takes a bit to display.

**Pull request merged**